### ISSUE TITLE
Color contrast ratio of "Remote" text is 4.381:1 which is less than 4.5:1: A11y_Azure Function_Default screen_Color Contrast

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode-azurefunctions#4481]
### GIT Hub Tags:
#A11yTCS; #A11ySev3; #Azure Tools for VS Code; #BM_Azure Tools for VS Code_Win32_Nov2024; #DesktopApp; #A11yMAS; #WCAG1.4.3; #ColorContrast; #Win11; #FTP;

### Environment and OS details:
Application name: Azure Tools for VS Code
URL: Visual studio code

### Reproduction Steps:

1. Open Visual studio code
2. TAB till extension button and press ENTER then search for "Azure Functions"
3. TAB till install and press ENTER.
4. TAB till "Azure" button and press ENTER.
5. TAB till "Resource" section.
6. Open accessibility insights for windows and check color contrast ratio of "Remote" text.
7. Observe the issue whether Color contrast ratio of "Remote" text is 4.381:1 which is less than 4.5:1 or not.

### Actual behavior:
Color contrast ratio of "Remote" text is 4.381:1 which is less than 4.5:1

### Expected behavior:
Color contrast ratio of "Remote" text should be greater than or equals to 4.5:1

### User Impact:
Low vision users will not be able to see "Remote" text if color contrast ratio will be less than 4.5:1.

### Attachment:

![Image](https://github.com/user-attachments/assets/7fd5b093-cc4f-4f74-93e9-ef41afc4000f)

**Comment by kupatkar99** [comment]
External Bug ID: https://github.com/microsoft/vscode/issues/246455

**Comment by keerthiduvvuri** [comment]
GithubTags:#ResolvedByTT
